README.md

```diff
@@ -1,8 +1,12 @@
+[![Maintainability](https://api.codeclimate.com/v1/badges/0f12a7e73736f8bbfb9d/maintainability)](https://codeclimate.com/github/Open-CMSIS-Pack/vscode-cmsis-debugger/maintainability)
```

Comment:
We probably want to hide the badges on the VS Code marketplace. Can we remove them in CI before packaging with `vsce`?

Reply:
Will open separate PR to remove this.